include/xtensor/xbuilder.hpp

```diff
@@ -247,6 +247,23 @@ namespace xt
             }
         };
 
+        template <class T, class S>
+        using both_integer = xtl::conjunction<std::is_integral<T>, std::is_integral<S>>;
+
+        template <class T, class S = T, std::enable_if_t<xtl::negation<both_integer<T, S>>::value, int> = 0>
```

Comment:
Why not using XTL_REQUIRES? I think the following is easier to read:
```cpp
template <class T, class S = T, XTL_REQUIRES(xtl::negation<both_integer<T, S>>)>
```

Reply:
You're right, thanks :)